test/unit/test_copy_expid.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+from conftest import AutosubmitExperimentFactory
+from autosubmit.autosubmit import Autosubmit
+
+from typing import Callable
+
+"""Test to check the functionality of the -y flag. It will copy the hpc too."""
+
+_EXPERIMENT_DESCRIPTION = "test descript"
+
+
+def test_create_expid_default_hpc(create_autosubmit_exp: AutosubmitExperimentFactory) -> None:
```

Comment:
Instead of `create_autosubmit_exp`, the fixture now is `autosubmit_exp`. This test can be moved to `test/integration/test_expid.py` (already exists). :+1: